v2/dtos/event_test.go

```diff
@@ -51,3 +57,18 @@ func TestFromEventModelToDTO(t *testing.T) {
 		})
 	}
 }
+
+func TestEvent_ToXML(t *testing.T) {
+	// Since the order in map is random we have to verify the individual items exists without depending on order
+	contains := []string{
+		"<Event><ApiVersion>v2</ApiVersion><ID>7a1707f0-166f-4c4b-bc9d-1d54c74e0137</ID><Pushed>1594963842</Pushed><DeviceName>TestDevice</DeviceName><Created>1594963842</Created><Origin>1594963842</Origin>",
+		"<GatewayID>Houston-0001</GatewayID>",
```

Comment:
Should we also add `<Tags>`  here?
There is only `</Tags>` here

Reply:
Yes! Great catch! Fixed. THX!